frontend/stylesheets/_drawings.less

```diff
@@ -58,3 +58,9 @@
 .drawing-upgrade-storage {
   background: url("/assets/images/pricing/upgrade_storage_light_mode.svg") right -10px / 25% no-repeat;
 }
+
+.drawing-license-expired {
+  width: 500px;
+  height: 385px;
```

Comment:
I did not manage to make the size of this drawing dynamic in a reasonable time. Thus, I went for a constant  sizing